galgebra/ga.py

```diff
@@ -1683,22 +1683,10 @@ def er_blade(self, er, blade, mode='*', left=True):
 
         .. math:: e^{j}*(e_{i_{1}}\wedge ...\wedge e_{i_{r}})
         """
-        if mode == '*':
-            base = self.blade_to_base_rep(blade)
-            if left:
-                return self.base_to_blade_rep(self.mul(er, base))
-            else:
-                return self.base_to_blade_rep(self.mul(base, er))
-        elif mode == '^':
-            if left:
-                return self.wedge(er, blade)
-            else:
-                return self.wedge(blade, er)
+        if left:
+            return self.Mul(er, blade, mode=mode)
         else:
-            if left:
-                return self._dot(er, blade, mode=mode)
-            else:
-                return self._dot(blade, er, mode=mode)
```

Comment:
Note that these lines do not work at all, since I removed `_dot` in a previous commit.